docs/src/Box.doc.js

```diff
@@ -651,23 +803,38 @@ function ButtonFlyoutExample() {
     </>
   );
 }`}
-  />,
-);
-
-card(
-  <Example
-    description={`
-It's possible to use box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.
-  `}
-    id="zindex"
-    name="ZIndex"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`It's possible to use Box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.`}
```

Comment:
We'll need to expand on this a lot more in the zIndex docs. I think our approach is pretty confusing.